app/views/users/index.html.erb

```diff
@@ -1,5 +1,7 @@
-<h1>Welcome to the Ruby on Rails Blog</h1>
-<h2>Posts</h2>
-<h3>Post number 1</h3>
-<h3>Post number 2</h3>
-<h3>Post number 3</h3>
+<h2>Our members</h2>
+<% @users = User.all; %>
```

Comment:
- [ ] the Users controller `#index` action should render all users to its corresponding Views index page.